src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -25,10 +25,11 @@ public LineService(LineRepository lineRepository, StationService stationService)
         this.stationService = stationService;
     }
 
+    @Transactional
     public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
-        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
+        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance(), request.getAdditionalFee()));
```

Comment:
```suggestion
        Line persistLine = lineRepository.save(request.toLine());
```

이건 취향일 수 있는데요, 저는 개인적으로 Request 에서 받은 값으로만 entity를 생성한다면 이런식으로 사용하고있습니다.
Request 객체는 엔티티를 알아도 상관없다고 생각해요 :)
엔티티가 변경되면 어짜피 변경되어야 하는 객체(엔티티와 강한 결합을 가진 객체)라고 생각하기 때문입니다.